src/temporal.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    HoD <: Transform
```

Comment:
when adding DoW or any other temporal transform later it might help to define a supertype `TemporalTransform` - just an idea

Reply:
yeah, out of scope for this MR though until we find more temporal transforms